### ISSUE TITLE
Awesomplete and app.js scripts not found

### ISSUE THREAD

**Issue by Lsquared13** [poanetwork/blockscout#2920]
Manual deployment on AWS runs successfully, but results in a page with improper styling and several console errors, most related to an inability to load awesomplete.

Hoping someone familiar with this codebase can point me in the right direction for further debugging.

### Environment

* Elixir & Erlang/OTP versions (`elixir -version`): Erlang/OTP 21 ; Elixir 1.9.4 (compiled with Erlang/OTP 20)
* Operating System: Amazon Linux 2 (EC2)
* Blockscout Version/branch: master

### Steps to reproduce

- Manually apply terraform infrastructure generated by ansible
- Manually deploy software using instructions output by terraform

### Expected behaviour

The app should load with proper styling and without 404 errors in the console

### Actual behaviour

The page styling is broken and several errors appear in the console

<img width="404" alt="Screen Shot 2019-12-12 at 6 57 18 PM" src="https://user-images.githubusercontent.com/4107447/70759896-dbf4e580-1d15-11ea-82ea-7c975459ab85.png">

**Comment by vbaranov** [comment]
@Lsquared13 which Node js version you used to build front-end assets?

**Comment by Lsquared13** [comment]
Made the following change to the build script to check this at build time:

<img width="232" alt="Screen Shot 2019-12-13 at 10 51 40 AM" src="https://user-images.githubusercontent.com/4107447/70812983-b3610000-1d96-11ea-9b4a-dfa5fdaeaf65.png">

Looks like npm version `3.10.10` and node version `v6.17.1`. Those versions seem relatively old so maybe updating them would help?

That said, I'm pretty sure I'm just using the versions provided by the AMI, which itself was selected by the generated terraform config.  Is it possible I missed a step that updates these versions somewhere?

<img width="297" alt="Screen Shot 2019-12-13 at 11 01 59 AM" src="https://user-images.githubusercontent.com/4107447/70813760-07b8af80-1d98-11ea-96dd-bc433f3cce08.png">

Thanks for the prompt response!

**Comment by vbaranov** [comment]
Your Node version is too old. You need to upgrade it to the latest LTS release. Please check Blockscout requirements: https://docs.blockscout.com/for-developers/information-and-settings/requirements

**Comment by vbaranov** [comment]
Am I correct that you used Blockscout deployment from AWS Marketplace?

**Comment by vbaranov** [comment]
>Looks like npm version 3.10.10 and node version v6.17.1. Those versions seem relatively old so maybe updating them would help?

Definitely, it should solve the issue. Node 12 should be used.

**Comment by Lsquared13** [comment]
> Definitely, it should solve the issue. Node 12 should be used.

Unfortunately I didn't have any luck with that. I did the `npm install` in `apps/explorer` with node `v12.13.1` and npm `6.12.1` and still have the same symptoms after deployment.  

My change to the [build script](https://github.com/poanetwork/blockscout/blob/master/bin/deployment/build) looks like this (3 lines under the comment):

<img width="412" alt="Screen Shot 2019-12-13 at 5 41 15 PM" src="https://user-images.githubusercontent.com/4107447/70836907-d0fe8b80-1dcf-11ea-824d-862cc4100bd6.png">

is it possible I'm missing some other dependencies I need to build?

 > Am I correct that you used Blockscout deployment from AWS Marketplace?

No, I am under the impression that the AWS Marketplace version isn't available per [this documentation](https://docs.blockscout.com/for-developers/aws-marketplace).  

I did an ansible deployment for infrastructure, copied the terraform output, applied it manually, and followed the instructions in the terraform output for deployment
